documentation/howtos.md

```diff
@@ -4,61 +4,164 @@ By default, instances are not provisioned with either ecommerce or the Course Di
 service. However, support is available to manually enable those services. To do so:
 
 * Ensure that the [`refresh_course_metadata` cron task](https://github.com/open-craft/configuration/blob/c1e576eabefea82d21d7785810126e39752cd14e/playbooks/roles/discovery/tasks/main.yml)
-  is added to the ansible discovery tasks.
+  is added to the ansible discovery tasks. If using a configuration revision
+  that doesn't include this, the following extra cronjob can be added to extra
+  configuration:
+
+```
+# must add as root cronjob, even though we want to run the task as the
+# discovery user, because these cronjobs are added before the discovery user is
+# created.
+EDXAPP_ADDITIONAL_CRON_JOBS:
+- name: "discovery: hourly course metadata refresh"
+  user: "root"
+  job: "sudo -u discovery bash -c 'source {{ discovery_home }}/discovery_env; {{ COMMON_BIN_DIR }}/manage.discovery refresh_course_metadata'"
+  hour: "*"
+  minute: "43"
+  day: "*"
+```
+
 * Set these extra variables on the instance:
 
-        SANDBOX_ENABLE_DISCOVERY: yes
-        SANDBOX_ENABLE_ECOMMERCE: yes
-        DISCOVERY_VERSION: 'open-release/ginkgo.1'
-        nginx_discovery_gunicorn_hosts:
-            - "127.0.0.1"
-        COMMON_HOSTNAME: "your-site-name"
-        ECOMMERCE_PAYMENT_PROCESSOR_CONFIG:
-            your-partner-code:
-              paypal:
-                ...
+```
+# prevent creating example partners; this isn't required, and will cause provision errors later
+ecommerce_create_demo_data: false
+
+SANDBOX_ENABLE_DISCOVERY: yes
+SANDBOX_ENABLE_ECOMMERCE: yes
+DISCOVERY_VERSION: "{{ ECOMMERCE_VERSION }}"
+nginx_discovery_gunicorn_hosts:
+    - "127.0.0.1"
+COMMON_HOSTNAME: ""
+ECOMMERCE_PAYMENT_PROCESSOR_CONFIG:
+    partn_id: # this is arbitrary, limited to 8 characters; will be used later
+      paypal:
+        ...
+
+# Have to build the full broker URLs to ensure the vhost path is included
+ECOMMERCE_BROKER_URL: '{{ EDXAPP_CELERY_BROKER_TRANSPORT }}://{{ EDXAPP_CELERY_USER }}:{{ EDXAPP_CELERY_PASSWORD }}@{{ EDXAPP_CELERY_BROKER_HOSTNAME }}{{ EDXAPP_CELERY_BROKER_VHOST }}'
+ECOMMERCE_WORKER_BROKER_URL: '{{ EDXAPP_CELERY_BROKER_TRANSPORT }}://{{ EDXAPP_CELERY_USER }}:{{ EDXAPP_CELERY_PASSWORD }}@{{ EDXAPP_CELERY_BROKER_HOSTNAME }}{{ EDXAPP_CELERY_BROKER_VHOST }}'
+ECOMMERCE_WORKER_ECOMMERCE_API_ROOT: '{{ ECOMMERCE_ECOMMERCE_URL_ROOT }}/api/v2/'
+```
```

Comment:
Changing the currency was pretty involved, so the details of that should be noted here.

> To change the default currency used in the ecommerce system from USD ($) to, for example, British points (£), add this extra configuration:

```
# Use a configuration branch which contains this fix:
# https://github.com/open-craft/configuration/pull/119/files
EDXAPP_PAID_COURSE_REGISTRATION_CURRENCY: ['gbp', '£']
ECOMMERCE_OSCAR_DEFAULT_CURRENCY: 'GBP'
ECOMMERCE_REPOS:
  - PROTOCOL: '{{ COMMON_GIT_PROTOCOL }}'
    DOMAIN: '{{ COMMON_GIT_MIRROR }}'
    PATH: 'open-craft'
    REPO: 'ecommerce.git'
    # Use a branch which contains this fix:
    # https://github.com/open-craft/ecommerce/pull/13/files
    VERSION: 'jill/set-default-currency'  # FIXME: change to opencraft-release/ironwood.2
    DESTINATION: "{{ ecommerce_code_dir }}"
    SSH_KEY: '{{ ECOMMERCE_GIT_IDENTITY }}'
```